package.json

```diff
@@ -35,7 +35,7 @@
     "start-js": "react-static start",
     "start": "npm-run-all -p watch-css start-js",
     "build-js": "react-static build",
-    "build": "npm-run-all build-css build-js && cp -r dist/* _dist/",
```

Comment:
will we still be able to test the static build locally without this?

Reply:
Yes by running `./scripts/serve-build.sh`, though I'm not sure that's a great option because things won't get automatically rebuilt.

I made this change because I didn't want to copy from the nonstandard `_dist` in the Dockerfile https://github.com/cityofaustin/janis/pull/127/files#diff-d92c0cc6cc3b84dda3c92547bcf56981R28

I'll spend some more time to make sure `./scripts/serve-local.sh` works well for development.